internal/rbd/rbd_util.go

```diff
@@ -174,7 +174,18 @@ var supportedFeatures = map[string]imageFeature{
 		needRbdNbd: false,
 	},
 	librbd.FeatureNameExclusiveLock: {
-		needRbdNbd: true,
+		needRbdNbd: false,
```

Comment:
reason for removing nbd here?

Reply:
The reason is:

```
  Warning  ProvisioningFailed    8s (x8 over 70s)   rbd.csi.ceph.com_ceph-csi-rbd-provisioner-6b9f54d9df-4slxv_a61dd25a-888b-4104-8e9b-e5ec09125b38  failed to provision volume with StorageClass "csi-rbd-sc": rpc error: code = InvalidArgument desc = feature exclusive-lock requires rbd-nbd for mount
```

In ancient times this feature was supported only by librbd, but currently support with krbd